monkey/monkey_island/cc/ui/src/components/pages/LandingPage.tsx

```diff
@@ -74,4 +89,41 @@ function MonkeyInfo() {
   );
 }
 
+function ScenarioInfo() {
+    // TODO change links when added to documentation
+    return (
+      <>
+        <div className={'scenario-info'}>
+          Check monkey documentation hub for better explanation
+          on <a href='https://www.guardicore.com/infectionmonkey/docs' rel="noopener noreferrer" target="_blank">
+           Ransomware
+          </a> and <a href='https://www.guardicore.com/infectionmonkey/docs' rel="noopener noreferrer" target="_blank">
+           Custom
+          </a> simulations.
```

Comment:
"Check the Infection Monkey documentation hub for more information on simulations."
And link "simulations" to the docs. Linking each simulation to its respective page doesn't make sense for the long run.